controller/lib/core/sensors.cpp

```diff
@@ -147,7 +146,8 @@ float pressure_delta_to_volumetric_flow(float diffPressureInKiloPascals) {
       1.0f / sqrtf(venturiPortArea * venturiPortArea -
                    venturiChokeArea * venturiChokeArea);
 
-  float sgn = copysignf(1.0f, diffPressureInKiloPascals);
-  return sgn * ROOT_OF_TWO_OVER_DENSITY_OF_AIR * venturiAreaProduct *
-         bernoulliAreaDivisor * sqrtf(abs(diffPressureInKiloPascals) * 1000.0f);
+  float sgn = copysignf(1.0f, delta.kPa());
+  return cubic_m_per_sec(sgn * ROOT_OF_TWO_OVER_DENSITY_OF_AIR *
```

Comment:
The ROOT_OF_TWO... thing should have units in its name too.

Reply:
Done; changed the constant from `sqrt(2/density-of-air)` to `density-of-air` to avoid the problem above of having a units word salad.